etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/ObrOverrides.java

```diff
@@ -0,0 +1,81 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.ServiceRequest;
+import org.hl7.fhir.r4.model.StringType;
+
+public class ObrOverrides implements CustomFhirTransformation {
```

Comment:
Missing javadocs to explain here what the transformation should do. You can take a look at other transformations for inspiration

Reply:
I think the name of the transformation should be more specific. What's the OBR value being overriding in this transformation?

Reply:
Added the javadoc, have not updated the name yet.

Reply:
Name updated as per our talk today!